processing/src/test/java/org/apache/druid/collections/spatial/search/RectangularBoundTest.java

```diff
@@ -48,6 +50,57 @@ public void testCacheKey()
     ));
   }
 
+  @Test
+  public void testRectangularBound()
+  {
+    float[][] insidePoints = new float[][]{
+        {37.795717853074635f, -122.40906979480418f},
+        {37.79625791859653f, -122.39638788940042f},
+        {37.79685798676811f, -122.39335030726777f},
+        {37.7966179600844f, -122.39798262002006f}
+    };
+    float[][] outsidePoints = new float[][]{
+        {37.79805810848854f, -122.39236309307468f},
+        {37.78197485768925f, -122.41886599718191f},
+        {37.798298130492945f, -122.39608413118715f},
+        {37.783595343766216f, -122.41932163450181f}
+    };
+    RectangularBound rectangularBound = new RectangularBound(
+        new float[]{37.78185482027019f, -122.41795472254213f},
+        new float[]{37.797638168104185f, -122.39228715352137f},
+        10
+    );
+    for (float[] insidePoint : insidePoints) {
+      Assert.assertTrue(rectangularBound.contains(insidePoint));
+    }
+    for (float[] outsidePoint : outsidePoints) {
+      Assert.assertFalse(rectangularBound.contains(outsidePoint));
+    }
+  }
+
+  @Test
+  public void testDeSerForDoubleArrays() throws JsonProcessingException
```

Comment:
nit: test name is stale i guess. Also instead of doing it with a string, it would be nicer to make a `RectangularBound` serialize it to string, and then deserialize back to an object to make sure round tripping works ok

Reply:
modified test